Makefile

```diff
@@ -3,4 +3,23 @@ run:
 run-debug:
 	cargo run -- --dev --tmp --enable-offchain-indexing=true -l sc_offchain=trace
 keystore-add:
-	curl http://localhost:9933 -H "Content-Type:application/json;charset=utf-8" -d "@scripts/keystore.json"
\ No newline at end of file
+	curl http://localhost:9933 -H "Content-Type:application/json;charset=utf-8" -d "@scripts/keystore.json"
+
+code-prettier:
```

Comment:
This target sounds a bit like `prettier`, which is a different code formatter/CLI. This is extremely minor, but maybe `code-fmt`, or `format-code` or something like that would sound less similar?

Reply:
Sure, have changed it to `format-code`. Thanks!